CapstoneProject/Assets/Scripts/Core/Audio/SoundManager.cs

```diff
@@ -118,7 +147,23 @@ public void ChangeSoundVolume(float soundValue)
     public void ChangeMusicVolume(float musicValue)
     {
         musicMod = musicValue;
-        Sound s = Array.Find(sounds, sound => sound.name == "ExampleSong");
-        s.source.volume = (s.volume * musicMod);
+        scene = SceneManager.GetActiveScene();
+        string checkScene= scene.ToString();
+        if (checkScene == "Level1")
+        {
+           Sound s = Array.Find(sounds, sound => sound.name == "Level1Music");
+            s.source.volume = (s.volume * musicMod);
```

Comment:
this is done in all ifs, maybe just do it once below them all?

Reply:
this would actually cause a bug where changing the volume within the main menu would not change the volume